### ISSUE TITLE
[Bug] Xenon Headlights not staying

### ISSUE THREAD

**Issue by BigSmashGG** [FrenzyRoleplay/server-issues#363]
**Describe the bug**
Going to Benny's the Xenon headlights in different colors being purchased for vehicle is not staying on the car when it is entered and removed from the garage

**To Reproduce**
1. Goto Bennys
2. Install Xenon Headlights in any color
3. Store vehicle and remove it from garage
4. Xenon Lights are no longer on the car, resets back to default

**Expected behavior**
Xenon lights in purchased color should stay on car

**Comment by Trauma-Llama** [comment]
I believe this is known and it's just a fivem thing not sure if it is fixable.

**Comment by WTFJamie** [comment]
yeah sadly this is something that has been around for a long time, there maybe a fix somewhere out there for it but i feel like the easiest solution for now until its found is that if your car has xenons installed that you dont have to pay for the color for them until a fix is found

**Comment by badcodestv** [comment]
Is this still happening?

**Comment by BigSmashGG** [comment]
yes, just tested it on the car. put the lights to black light, took the car to legion garage and parked it. pulled it out and they are back to standard white.

**Comment by badcodestv** [comment]
Was it xenon lights or xenon + colored lights

**Comment by BigSmashGG** [comment]
Xenon headlights. Changing them to any color, storing the car in garage and bringing it back out the headlights go back to the default headlights

**Comment by cademullen** [comment]
@BigSmashGG Has this been resolved?

**Comment by BigSmashGG** [comment]
i dont believe so no. no one has followed up with me on it in 27 days.

**Comment by cademullen** [comment]
@BigSmashGG What cars do they not stick on for you?

**Comment by BigSmashGG** [comment]
all of the cars. I haven't checked today but when I changed the color of
the lights, saved the car, brought it back out from garage the default
lights were back on the car

**Comment by cademullen** [comment]
@BigSmashGG Public Garages, or house garages?

**Comment by cademullen** [comment]
@BigSmashGG Where did you get it changed at?

**Comment by cademullen** [comment]
@BigSmashGG Let me know what car it is so i can test that specific car.

**Comment by BigSmashGG** [comment]
I tried it at public garage. legion

**Comment by BigSmashGG** [comment]
i just tested the changing of the light colors on 2 cars, i put them in the legion garage and pulled them back out and the lights are saving the xenon color lights. looks like it is working now

**Comment by BigSmashGG** [comment]
the lights are not staying custom. after driving around for a while they
reverted back to default lights

**Comment by BorsaMetin** [comment]
@BorsaMetin is here!

**Comment by BigSmashGG** [comment]
the headlights are not staying on the cars still. any update on this one?
